_drafts/2015-02-05-play2-californium-raspi.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: post
+title: Integrating Californium with the Raspberry Pi
+author: lucas
+---
+
+On our Raspberry Pi web interface we want to display the data from all our plant nodes in "realtime" (in double-quotes because, well, we're talking about websites). In this post we will be introducing our technology stack that will be running on the Raspberry Pi and explain step by step how all the piece fit and can be made to work together. 
```

Comment:
the text in () seems like unnecessary overhead to me. I think it's obvious that web pages can't fulfill hard real time requirements ;)